mhkit/tests/Utils_TestGenUtils.m

```diff
@@ -71,6 +73,142 @@ function test_magnitude_phase(testCase)
             assert(all(phase1 == theta))
             assert(all(phase2 == phi))
         end
+        function test_read_nc_file_group(testCase)
+            % Check MATLAB version
+            if isMATLABReleaseOlderThan("R2021b")
+                return;
+            end
+            fnms = {dir('example_ncfiles/').name};
```

Comment:
similar to other comments, if only testing the 2 files specified, you can remove this variable